### PR TITLE
Add Feed The Beast.app (latest)

### PR DESCRIPTION
**Pull request by izeau**

### CODE REVIEW

File: Casks/feed-the-beast.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'feed-the-beast' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://ftb.cursecdn.com/FTB2/launcher/FTB_Launcher.zip'
```
Comment (vitorgalvao) [inline Casks/feed-the-beast.rb]: Please add a `# cursecdn.com is the official download host per the vendor homepage` above this line.

File: Casks/feed-the-beast.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name 'Feed the Beast'
+  homepage 'http://www.feed-the-beast.com/'
+  license :apache
+  depends_on :cask => 'java'
```
Comment (vitorgalvao) [inline Casks/feed-the-beast.rb]: `java` is a weird case. We actually have a `depends_on :java` special case (not yet functional). In the mean time, please remove this line and add [this caveat](https://github.com/caskroom/homebrew-cask/blob/ac7fa0d912be1f242b1cf10ba14f96a1f3223f68/Casks/eid-mw.rb#L13-16).

**Comment by izeau** [comment]
This should be fixed now, thanks!

**Comment by vitorgalvao** [comment]
Looks good, thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by izeau** [comment]
Done! :wink:

**Comment by vitorgalvao** [comment]
Thank you. Merged.

**Pull request merged**